actix-web-codegen/src/route.rs

```diff
@@ -286,7 +305,13 @@ impl ToTokens for Route {
             }
         };
 
+        let doc_comment_token_stream: TokenStream2 = doc_attribute
+            .iter()
+            .map(ToTokens::to_token_stream)
+            .collect();
+
```

Comment:
This is not needed, you can just use a repetition in the output. Suggestion in next review comment.